README.md

```diff
@@ -1,4 +1,14 @@
-# Mirantis Hybrid Cloud Platform
+# Mirantis Hybrid Multi Cluster
+
+## Overview
+
+Mirantis Hybribd Multi Cluster is part of Mirantis Project 2A which is focused on delivering a 
```

Comment:
`Hybrid`

Reply:
Good catch, thats what I get for multi-threading